### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by duncandewhurst**
to support publishers already working on implementing OC4IDS based on the alpha-2 release

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+In addition to the specific changes to schema and codelists noted below:
+
+* Various refinements and clarifications were made to schema and codelist descriptions.
+* Guidance on mapping values from OCDS was moved from the schema to the IDS and OCDS mapping section of the documentation.
+* Documentation was expanded and restructured.
+
+### Packaging
+
+* Add [project package schema](http://standard.open-contracting.org/infrastructure/latest/en/reference/package/), OC4IDS data must be published as part of a project package.
```
Comment (jpmckinney) [inline changelog.md]: Use period not comma

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+### Schema updates
+
+* `sector` - use projectSector open codelist
+* `contractingProcess` - add required `id` field
+* `linkedRelease` - make `id` a required field
+* `variations` - renamed to `modifications`
```
Comment (jpmckinney) [inline changelog.md]: Use present tense

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+### Packaging
+
+* Add [project package schema](http://standard.open-contracting.org/infrastructure/latest/en/reference/package/), OC4IDS data must be published as part of a project package.
+
+### Schema updates
+
+* `sector` - use projectSector open codelist
+* `contractingProcess` - add required `id` field
+* `linkedRelease` - make `id` a required field
```
Comment (jpmckinney) [inline changelog.md]: Capitalize LinkedRelease. Change to "make `id` required".

File: changelog.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+* Documentation was expanded and restructured.
+
+### Packaging
+
+* Add [project package schema](http://standard.open-contracting.org/infrastructure/latest/en/reference/package/), OC4IDS data must be published as part of a project package.
+
+### Schema updates
+
+* `sector` - use projectSector open codelist
+* `contractingProcess` - add required `id` field
```
Comment (jpmckinney) [inline changelog.md]: Capitalize ContractingProcess

File: changelog.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+* `linkedRelease` - make `id` a required field
+* `variations` - renamed to `modifications`
+* `location` - add required `id` field
```
Comment (jpmckinney) [inline changelog.md]: Capitalize Location

File: changelog.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+### New codelists
+
+* projectSector codelist - add codelist for project sector
```
Comment (jpmckinney) [inline changelog.md]: Use backticks for field name.

File: changelog.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+ * lender
+ * equityInvestor
+ * consortiaMember
+ * interestedParty
+ * grantor
+ * disqualifiedBidder
+ * socialWitness
+ * otherWitness
+ * notary
+* documentType codelist - remove PPP specific codes:
```
Comment (jpmckinney) [inline changelog.md]: Hyphenate PPP-specific

File: changelog.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+* variationType codelist - rename codelist to modificationType
+* partyRole codelist - add OC4IDS codes mentioned in schema and mapping:
+ * funder
+ * administrativeEntity
+* partyRole codelist - add codes from OCDS partyRole codelist:
+ * buyer
+ * procuringEntity
+ * supplier
+ * tenderer
+* partyRole codelist - remove PPP specific codes:
```
Comment (jpmckinney) [inline changelog.md]: Hyphenate PPP-specific

File: changelog.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+* `location` - add required `id` field
+
+### New codelists
+
+* projectSector codelist - add codelist for project sector
+
+### Codelist updates
+
+* projectStatus codelist - replace 'construction' with 'implementation'
+* variationType codelist - rename codelist to modificationType
```
Comment (jpmckinney) [inline changelog.md]: Remove second "codelist"

**Review by jpmckinney — COMMENTED** [review]
Small copy-edits.

File: changelog.md (hunk 1 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 2 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 3 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 4 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 5 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 6 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 7 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 8 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

File: changelog.md (hunk 9 above)
Comment (duncandewhurst) [reply changelog.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769

**Comment by duncandewhurst** [comment]
@jpmckinney I've made those updates to the changelog and made it discoverable from the docs.

Could you review and approve so we can include it in the release to fix the schema urls (see https://github.com/open-contracting/infrastructure/issues/113#issuecomment-502630050)?

Thanks

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**